### PR TITLE
Fix GitHub API authenthication

### PR DESCRIPTION
**Pull request by s-ludwig**
Uses authorization tokens instead of the password for accessing the GitHub API.

### CODE REVIEW

**Comment by Geod24** [comment]
Didn't knew we had Cirrus CI here

File: source/app.d [hunk 1]
```diff
@@ -85,7 +85,7 @@ void defaultInit(UserManController userMan, DubRegistry registry)
 
 struct AppConfig
 {
-	string ghuser, ghpassword,
+	string ghuser, ghauth,
```
Comment (Geod24) [inline source/app.d:88]: Do we still need `ghuser` then ?

Comment (s-ludwig) [reply source/app.d:88]: No, you are right, I'll remove it.

**Comment by s-ludwig** [comment]
BTW, I've deployed this version in the meantime and updates are working.

**Review by Geod24 — APPROVED** [review]
LGTM. Do you know what's up with the CI ? Thanks for the quick reaction.

**Comment by s-ludwig** [comment]
No idea what the docker task is supposed to do, but I noticed it failing since quite a while.

**Pull request merged**